src/feature/hs/hs_ob.c

```diff
@@ -285,14 +285,17 @@ compute_subcredentials(const hs_service_t *service,
   const uint64_t tp = hs_get_time_period_num(0);
 
   tor_assert(service);
-  tor_assert(subcredentials);
+  tor_assert(subcredentials_out);
   /* Our caller has checked these too */
   tor_assert(service->desc_current);
   tor_assert(service->desc_next);
 
   /* Our caller made sure that we are an OB instance */
   num_pkeys = smartlist_len(service->config.ob_master_pubkeys);
-  tor_assert(num_pkeys > 0);
+  if (!num_pkeys) {
+    subcredentials_out = NULL;
```

Comment:
Should this be *subcredentials_out ?

Reply:
!Yes indeed. See 93cb807 for a unittest and fix